data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1701 @@
+			`	The waiter arrives, and they suggest some dishes, seeing as you didn't pick anything from the menu. The smell coming from the other tables, however, tells you to decline, so you politely leave the establishment. It seems the only way to find out what these Arachi want is to head to <destination>.`
+				accept
+
+mission "House Bebliss 2-FW"
+	name "Old Friends"
+	description "Go to <destination> and look for Freya to see if she is willing to let the members of House Bebliss study the Pug artifacts with her."
+	landing
+	to offer
+		not "Heliarch License: done"
+		not "Heliarch Investigation 2: active"
+		or
+			has "House Bebliss 1-A: done"
+			has "House Bebliss 1-B: done"
+	source "Factory of Eblumab"
+	destination "Pugglemug"
+	on offer
+		conversation
+			`You follow the coordinates given to you and land on top of a mountain, on a set of private hangars close to an estate covered by a dome similar to the one over the spaceport city. You are received by who you assume are Arachi butlers and maids, given the well-kept uniforms, and follow one of them into the mansion.`
+			`	They bring you to a large, wide room where three Arachi are preoccupied with some schematics and blueprints scattered over a table.`
+			`	"Captain <first> <last>!" One of them comes to greet you, after the servant announces your arrival. "A rare opportunity, it is, to such an exquisite guest entertain. For a drink, would you care?" After asking that, a maids in the room brings a platter with several cups, and you're surprised to discover familiar smells like coffee or tea. There are also some alcoholic beverages there and plain water too.`
+			`	"Difficult to replicate, some of these human drinks were, but for a special guest, the least we could do, it is," another one approaches, gesturing for you to pick one of the drinks, while signalling for another maid to bring him and the other two some drinks of their own.`
+			choice
+				`	"Thank you, but why exactly did you want to talk to me so badly?"`
+				`	"You asked me to come here for a job. What is the job?"`
+			branch resistance
+				has "Resistance: Questions: done"
+			`	"The great House Bebliss, we represent. Maintain the local hyperspace communication network, our House does," he says as he swirls the drink in his cup before taking a sip.`
+			`	The other one jumps in as gulps down his drink, "Studied hyperspace links, for millennia we have. More recently, an application of such phenomena that would help the Coalition, we have searched for."`
+				goto choice
+			label resistance
+			`	"To be trusted, you are. Assured us of that, Chiree has," the one Arach still by the blueprints says. "The Lunarium's guardian angels, we are," she boasts. "Safeguard them against the Heliarchs, our meddling in the communication network does. To better serve that purpose, look further into this field, we must."`
```

Comment:
`Do Arach have a concept of Christian guardian angels?`

My idea for the most practiced Arach religion would be something akin to a human monotheistic religion, funnily enough.
That wasn't the intention for this though, I just used the term because I felt it was the easier way to describe what they are boasting about.